packages/preset-uno/src/rules/static.ts

```diff
@@ -84,14 +84,67 @@ export const textDecorations: Rule[] = [
   ['no-underline', { 'text-decoration': 'none' }],
 ]
 
+export const textDecorationStyles: Rule[] = [
+  ['underline-solid', { 'text-decoration-style': 'solid' }],
+  ['underline-double', { 'text-decoration-style': 'double' }],
+  ['underline-dotted', { 'text-decoration-style': 'dotted' }],
+  ['underline-dashed', { 'text-decoration-style': 'dashed' }],
+]
+
 export const textAligns: Rule[] = [
   ['text-center', { 'text-align': 'center' }],
   ['text-left', { 'text-align': 'left' }],
   ['text-right', { 'text-align': 'right' }],
   ['text-justify', { 'text-align': 'justify' }],
 ]
 
-export const listStyleType: Rule[] = [
+export const fontStyles: Rule[] = [
+  ['italic', { 'font-style': 'italic' }],
+  ['not-italic', { 'font-style': 'normal' }],
+]
+
+export const fontSmoothings: Rule[] = [
+  ['antialiased', {
+    '-webkit-font-smoothing': 'antialiased',
+    '-moz-osx-font-smoothing': 'grayscale',
+  }],
+  ['subpixel-antialiased', {
+    '-webkit-font-smoothing': 'auto',
+    '-moz-osx-font-smoothing': 'auto',
```

Comment:
Could we add a unprefixed one here as well?

Reply:
Looks like there is no browser support unprefixed ones.

https://caniuse.com/?search=font-smoothing

Reply:
Yes, but in the future right? 👀